en/simulation/multi_vehicle_simulation_gazebo.md

```diff
@@ -8,12 +8,13 @@ A different approach is used for simulation with and without ROS.
 
 To simulate multiple iris or plane vehicles in Gazebo use the following commands in the terminal (from the root of the *Firmware* tree):
 ```
-Tools/gazebo_sitl_multiple_run.sh [-m <model>] [-n <number_of_vehicles>] [-w <world>]
+Tools/gazebo_sitl_multiple_run.sh [-m <model>] [-n <number_of_vehicles>] [-w <world>] [-s <script>]
 ```
 
 - `<model>`: The [vehicle type/model](../simulation/gazebo_vehicles.md) to spawn, e.g.: `iris` (default), `plane`, `standard_vtol`.
 - `<number_of_vehicles>`: The number of vehicles to spawn. Default is 3. Maximum is 255.
 - `<world>`: The [world](../simulation/gazebo_worlds.md) that the vehicle should be spawned into, e.g.: `empty` (default)
+- `<script>: This overrides the `-n` and `-s` allows spawning a heterogeneous script, e.g.: `-s "iris:3,plane:2,standard_vtol:3"`
```

Comment:
@Jaeyoung-Lim What is a script? What is its format?

I presume the number `iris:3` indicates how many of that "script" to create?

Reply:
@hamishwillee No, this is not part of the make command. As written in the doc, only `iris`, `plane`, and `standard_vtol` is supported. I meant script as specifying arbitrary number of vehicle types and numbers to spawn. The IDs are iteratively allocated from the first vehicle. ""iris:3,plane:2,standard_vtol:3"` Yes, it overrides `m` and `n`

Reply:
OK, tried to capture that. Please check updated version. FMI Can you do something like this to create whatever pattern you like?
```
"iris:1,plane:2,standard_vtol:3,iris:3,"
```

Reply:
Yes this is possible. The limit is the same as the other option (`-n`) where the total number of vehicles should not exceed 255

Reply:
Great. I'm OK with merging as soon as you are.